vignettes/openxlsx2.Rmd

```diff
@@ -30,107 +30,108 @@ For this example we will use example data provided by the package. You can locat
 ### Basic import
 We begin with the `openxlsx2_example.xlsx` file by telling R where to find this file on our system
 ```{r}
-xlsxFile <- system.file("extdata", "openxlsx2_example.xlsx", package = "openxlsx2")
+xlsx_file <- system.file("extdata", "openxlsx2_example.xlsx", package = "openxlsx2")
 ```
 
 The object contains a path to the xlsx file and we pass this file to our function to read the workbook into R
 ```{r}
 # import workbook
-wb_to_df(xlsxFile)
+library(openxlsx2)
+wb_to_df(xlsx_file)
```

Comment:
Currently, we couldn't see the` library(openxlsx2)` call. https://janmarvin.github.io/openxlsx2/articles/openxlsx2_basic_manual.html